braintree.gemspec

```diff
@@ -10,7 +10,7 @@ Gem::Specification.new do |s|
   s.author = "Braintree"
   s.email = "code@getbraintree.com"
   s.homepage = "https://www.braintreepayments.com/"
-  s.files = Dir.glob ["README.rdoc", "LICENSE", "lib/**/*.{rb,crt}", "spec/**/*", "*.gemspec"]
```

Comment:
We need spec files to be able to run our internal CI processes. I understand that this means a hefty chunk of data added to the gem size, but for now we need to keep it as is.

That said, I'm going to make a card for us to look into updating our CI processes so that we may _not_ need to include spec files future versions.

Reply:
@hollabaq86 Thanks for the review.

Would you like me to leave this pull request open?